uint32 ref_idx
uint32 mov_idx
geometry_msgs/Pose T
float64[] cov
float64[] cov_3d
float64 score

#string first_name
#string last_name
#uint8 age
#uint32 score